sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -843,20 +846,100 @@ static const int ZeMaxCommandListCacheSize = [] {
   return CommandListCacheSizeValue;
 }();
 
-static const pi_uint32 ZeCommandListBatchSize = [] {
+// Configuration of the command-list batching.
+typedef struct {
```

Comment:
I'm trying to build on Windows today, and I'm getting this error on this struct:
```
C:\iusers\cperkins\sycl_workspace\llvm\sycl\plugins\level_zero\pi_level_zero.cpp(850): error C2220: the following warning is treated as an error
C:\iusers\cperkins\sycl_workspace\llvm\sycl\plugins\level_zero\pi_level_zero.cpp(850): warning C5208: unnamed class used in typedef name cannot declare members other than non-static data members, member enumerations, or member classes
```

But the CI was able to build and pass.  I tried building from scratch with our python script and no luck.  MSVC 2019 I think on the Win dev machine.

Reply:
Would you type-name the struct to avoid possible errors? (since you can verify this)

Reply:
@smaslov-intel  - yeah, that was it. I opened a PR: https://github.com/intel/llvm/pull/4555